packages/labs/test-projects/test-elements-react/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@lit-internal/test-elements-react",
+  "type": "module",
```

Comment:
Add private: true please.

Is it also worth documenting in a `description` field that this is essentially the output of the react generator checked in for testing purposes?

Reply:
Thanks for catching that! Description vs README... I guess it's simple enough that a description would be fine.